src/content/effects.js

```diff
@@ -43,11 +42,14 @@ const effects = {
   },
   moveTo: (id) => {
     renderScene(id);
+    changeLocation(id);
   },
   changeNeeds: ({
     ...needs
   }) => {
-    Object.entries(needs).forEach(([key, value]) => { playerStatus[key] += value; });
+    Object.entries(needs).forEach(([key, value]) => {
+      playerStatus[key] += value;
```

Comment:
Store min and max values in constants. Replace code below with checking only modified values.